Docs 3303: Restructure ready-made integration manuals (#2135)
MultiSafepay/docs: content/integration/ready-made/lightspeed/_index.md

{{< /details >}}

{{< details title="Payment methods logos in your website footer" >}}

By default, the app does **not** support adding payment methods logos to your website footer. We provide a script for this, or you can ask your developer to add the logos to your theme. Themes can differ and you may need to make some changes for it to function.

**Adding logos via our script**

1. Sign in to your Lightspeed app.
2. Go to **Settings** > **Storefront payment icons**.
3. Click **Copy to clipboard**. 
5. In your **Lightspeed admin area**, go to **Settings** > **Web extras and custom Javascript**. 
6. Paste the script into the **Javascript textbox**, and set the status to **Enable**.
7. Click **Save**. The logos appear in the footer.

**Display order**  

Depending on the storefront, the display order of the logos is determined by the settings at the time of generation. If you update these settings, you must update the script as well.

**Size**  

By default the logos are 16 px high. In most themes, footer logos are found in the "div.payment-methods p". If needed, you can change the selector based on the theme.

**Resizing logos**

> 1. In the JavaScript for displaying the logos, locate the following `img` element near the end of the script:
> 

Author: ```suggestion
1. In the JavaScript for displaying the logos, locate the following `img` element near the end of the script:
    ```<img src="${msplt[e]}" alt="${e}" />```
2. Specify the height and width in pixels as required, e.g.:
    ```<img height="16" width="37" src="${msplt[e]}" alt="${e}" />```
```

The logos won't appear instantly. It takes a little time.

**JavaScript**

For the best user experience, we provide some Javascript and images, e.g. to add a dropdown for iDEAL and MultiSafepay icons for other payment methods. 

Some user-added themes or scripts may cause issues, e.g. missing images for payment methods. 

For assistance, ask your developer. 

All payment methods still work if you don't use the Javascript files. 

{{< /details >}}

{{< details title="Ordering payment methods in your checkout" >}}

To change the order in which payment methods appear on your checkout page, follow these steps:

1. Go to **Settings** > **Payment method settings**. 
2. Drag and drop the payment methods to the preferred order.
3. Click **Save**.

**Setting payment method order per language**

To set the payment method order for different languages, under the **Payment method settings** select a country / store language and set the order per language.

If no specific rule is set for a country, the **Default** order is used.

{{< /details >}}

{{< details title="Disabling payment methods" >}}

> You can disable payment methods in the **Payment method settings list**. Enabled payment methods show a green dot, when disabled its grey.

Author: ```suggestion
You can disable payment methods in the **Payment method settings list**. The dot next to the payment methods is green when enabled, and grey when disabled.
```

1. In the JavaScript for displaying the logos, locate the following `img` element near the end of the script:

    ```<img src="${msplt[e]}" alt="${e}" />```

2. Specify the height and width in pixels as required, e.g.:

>     ```<img height="16" width="37" src="${msplt[e]}" alt="${e}" />```

Reviewer: Oops! Did you mean to delete this?
